src/shared/components/community/sidebar.tsx

```diff
@@ -130,9 +130,13 @@ export class Sidebar extends Component<SidebarProps, SidebarState> {
             </a>
           )}
           {subscribed == SubscribedType.Pending && (
-            <div class="badge badge-warning mr-2">
+            <a
+              class="badge badge-warning mr-2"
```

Comment:
Could you turn this into a `btn btn-warning`

Reply:
I changed the class, but dont understand what you mean by removing <a> tag. The previous item also uses <a>.